lib/util/package-rules.ts

```diff
@@ -1,30 +1,17 @@
 import minimatch from 'minimatch';
-import { PackageRule, UpdateType, mergeChildConfig } from '../config';
+import {
+  PackageRule,
+  PackageRuleInputConfig,
```

Comment:
We should add seperate type import and remove any `export * from ...` as it would add runtime deps for type only imports 😕

Reply:
You mean, remove `export * from './types'` in `lib/config/index.ts`, then fix all broken imports in project? Can do this in separate PR

Reply:
yes, do this in seperate pr 🙃